docs/products/kafka/kafka-mirrormaker/troubleshooting/topic-not-replicated.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Why are some topics or partitions not replicated
+---
+
+Apache Kafka® MirrorMaker 2 stores its configs, states and offsets to ensure a resilient message replication across different Kafka clusters.
+There are a number of factors that may prevent target topics (or their partitions) from progressing as you would expect.
+
+The following guideline assumes that the user has previously [setup a MirrorMaker replication flow](/docs/products/kafka/kafka-mirrormaker/howto/setup-replication-flow)
+and has assessed an issue with the replication of some topics or partitions.
+
+:::note
+There are different ways of performing this issue assesment, including basic monitoring
+and [offset sync status analysis](/docs/products/kafka/kafka-mirrormaker/howto/log-analysis-offset-sync-tool).
+:::
+
+## RecordTooLargeException is observed in the logs
+
+This can be fixed by increasing the value of broker config message_max_bytes at target, as well as integration config [producer_max_request_size](https://registry.terraform.io/providers/aiven/aiven/latest/docs/resources/service_integration#producer_max_request_size-1). Following to this change, it is necessary that the workers restart their task.
+
+## Configuration issue
+
+The following two worker parameters may significantly impact how topic partitions are replicated:
+
+- [kafka_mirrormaker.offset_lag_max](/docs/products/kafka/kafka-mirrormaker/reference/advanced-params#kafka_mirrormaker_offset_lag_max) (100 by default) defines how out-of-sync a remote partition can be before it is re-synced. Given that this parameter is global to all replication topics, you need to find a compromise between:
```

Comment:
The offset sync is emitted also periodically from Kafka 3.7 onwards, currently the MM2 version run in Aiven: https://issues.apache.org/jira/browse/KAFKA-15906

Reply:
Added, Thanks!